### ISSUE TITLE
Gigaword Validation Data

### ISSUE THREAD

**Issue by stylelohan** [lancopku/Global-Encoding#25]
Hi,

I have downloaded Gigaword dataset from [havardnlp](https://github.com/harvardnlp/sent-summary), and there are 4 directories: **train**, **Giga**, **DUC2004**, and **DUC2003**.

I use  valid.title.filter.txt, valid.article.filter.txt, train.title.txt and train.article.txt in **train** as the tgt/src of valid/train data, and use two txt files in **Giga** as test data. However, the valid data has wrong size (189651) after preprocess. The weird thing is that when I run preprocess.py, the result shows "(0 and 0 ignored due to length == 0 or > )".

Would you know any method to fix this?
Thanks a lot!

**Comment by JustinLin610** [comment]
This problem may stem from empty lines in your dataset. Check if you have this issue.

**Comment by jiahuanluo** [comment]
@JustinLin610 Thanks for your job. 
I wonder how to split the data into `validation set` and `test set`. There are 18,691 lines in the `valid.article.filter.txt`. 
How could I get the 8k `validation set` and the 2k `test set` identical to that of your paper?
Thanks!
